xbmc/interfaces/json-rpc/schema/types.json

```diff
@@ -505,7 +505,7 @@
       "description": "Requesting the songgenres, artistid and/or sourceid fields will result in increased response times",
       "enum": [ "title", "description", "artist", "genre",
                 "theme", "mood", "style", "type", "albumlabel",
-                "rating", "votes", "userrating","year", "musicbrainzalbumid",
+                "rating", "votes", "userrating","year", "boxset", "musicbrainzalbumid",
```

Comment:
New fields must be added at the end, in case any existing API consumers are using position of fields rather than name.